### PR TITLE
refactor the Care card component and make it a variant of the Card component

### PR DESCRIPTION
**Pull request by chrimesdev**
## Description

Note: This is an early proof of concept

Refactor the Care card component and make it a variant of the Card component

dependant on #740 

## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] Follows our [coding standards and style guide](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [ ] CHANGELOG entry

### CODE REVIEW

**Comment by chrimesdev** [comment]
waiting on #740

**Comment by chrimesdev** [comment]
Dependant on https://github.com/nhsuk/nhsuk-service-manual/pull/1220

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,315 @@
...
+      <a class="nhsuk-u-font-size-22 nhsuk-u-padding-2 nhsuk-u-display-block" href="#Y">Y</a>
+    </li>
+    <li class="nhsuk-u-margin-bottom-0 nhsuk-u-float-left nhsuk-u-margin-right-1">
+      <a class="nhsuk-u-font-size-22 nhsuk-u-padding-2 nhsuk-u-display-block" href="#Z">Z</a>
+    </li>
+  </ol>
+</nav>
+```
+
+The new code uses a new utility class:
```
Comment (henocookie) [inline CHANGELOG.md:197]: We will need to add documentation to the styles section of the design system on the service manual website for the new utility classes

File: app/components/all.njk [hunk 2]
```diff
@@ -180,50 +176,41 @@
...
-      <li>makes your chest feel tight or heavy</li>
-      <li>also started with shortness of breath, sweating and feeling or being sick</li>
-    </ul>
-    <p>You could be having a <a href=\"https://www.nhs.uk\">heart attack</a>. Call 999 immediately as you need immediate treatment in hospital.</p>
-    "
-  }) }}
-
-  {{ careCard({
+  {{ card({
     "type": "immediate",
```
Comment (henocookie) [inline app/components/all.njk:210]: This has been renamed to a "Emergency" variant of the card

File: app/pages/examples.njk [hunk 3]
```diff
@@ -33,9 +33,9 @@
     <li><a href="../components/card/card-with-image.html">Card - Card with an image</a></li>
     <li><a href="../components/card/card-group.html">Card - Card group</a></li>
     <li><a href="../components/card/feature-card.html">Card - Feature card</a></li>
-    <li><a href="../components/care-card/care-card-non-urgent.html">Care card non-urgent (blue)</a></li>
```
Comment (henocookie) [inline app/pages/examples.njk:36]: Won't this need to be ".../card/care-card-non-urgent.html" with the files changes in this PR?

File: app/pages/examples.njk [hunk 4]
```diff
@@ -33,9 +33,9 @@
...
-    <li><a href="../components/care-card/care-card-urgent.html">Care card urgent (red)</a></li>
```
Comment (henocookie) [inline app/pages/examples.njk:37]: Won't this need to be ".../card/care-card-urgent.html" with the files changes in this PR?

File: app/pages/examples.njk [hunk 5]
```diff
@@ -33,9 +33,9 @@
...
-    <li><a href="../components/care-card/care-card-immediate.html">Care card immediate (red and black)</a></li>
```
Comment (henocookie) [inline app/pages/examples.njk:38]: Won't this need to be ".../card/care-card-immediate.html" with the files changes in this PR?

File: packages/components/card/README.md [hunk 6]
```diff
@@ -379,6 +379,149 @@ Find out more about the card component and when to use it in the [NHS digital se
 }) }}
 ```
 
+### Care card non-urgent (blue)
+
+[Preview the care card non-urgent (blue) component](https://nhsuk.github.io/nhsuk-frontend/components/care-card/care-card-non-urgent.html)
```
Comment (henocookie) [inline packages/components/card/README.md:384]: Won't this need to be "...components/card/care-card-non-urgent.html" with the file name changes in this PR?

**Review by sarawilcox — COMMENTED** [review]
Hi Dom, a couple of minor things.

Lower case "card" in Care Card refactor in line 7.

Upper case "care" at the start of the sentence in line 9.

As I mentioned, note that we now call the "immediate" card card "emergency" care card.

File: app/pages/examples.njk (hunk 5 above)
Comment (DomBaker) [reply app/pages/examples.njk:38]: This has been added.

File: CHANGELOG.md (hunk 1 above)
Comment (DomBaker) [reply CHANGELOG.md:197]: This will need to be added agreed.

File: app/components/card/care-card-emergency.njk [hunk 7]
```diff
@@ -22,6 +24,7 @@
           <p>You could be having a heart attack. Call 999 immediately as you need immediate treatment in hospital.</p>
```
Comment (DomBaker) [inline app/components/card/care-card-emergency.njk:24]: should this now say
```suggestion
          <p>You could be having a heart attack. Call 999 immediately as you need emergency treatment in hospital.</p>
```

Comment (sarawilcox) [reply app/components/card/care-card-emergency.njk:24]: Good spot, but I think, for the public, leave it as it is, @DomBaker.

File: CHANGELOG.md (hunk 1 above)
Comment (henocookie) [reply CHANGELOG.md:197]: Created an issue for me to work on this 😄  https://github.com/nhsuk/nhsuk-service-manual/issues/1626

**Review by DomBaker — APPROVED** [review]

**Pull request merged**